titiler/stacapi/settings.py

```diff
@@ -67,7 +68,7 @@ class RetrySettings(BaseSettings):
     retry_factor: Annotated[float, Field(ge=0.0)] = 0.0
 
     model_config = {
-        "env_prefix": "TITILER_STACAPI_API_",
+        "env_prefix": "TITILER_STACAPI_RETRY_",
```

Comment:
some of these `env_prefix`s were the same, which seems like it could cause a bug if one of these classes both had the same attribute name so I thought we could rename the env_prefixes to match the class name. What do you think?

Reply:
It's not necessary, pydantic basettings is smart enough to handle this. 

they are distributed into multiple classes because they are not used at the same places

Reply:
ok reverted those changes